client/README.md

```diff
@@ -1,9 +1,15 @@
 # BitBake recipe language support in Visual Studio Code
 
-## Set BitBake's path
-Some features require to know where your BitBake's folder is located. The extension will by default assume it is located at the root of the project in a folder named `bitbake`. If your BitBake folder is located somewhere else, set its path in the settings in order to have full features.
+## Setup the extension
+In order to work properly, the extension needs to know your Bitbake's location and build folder. It will make the following assumptions:
```

Comment:
Maybe we can be more specific than "work properly". The extension highlighting works without for instance. Maybe just "For all the features to be enabled,"

Reply:
I'd also rephrase "it will make the following assumptions" -> "It will look for it at these default locations"

Reply:
Lets remove it it in PR #16